packages/shared/src/components/auth/OnboardingSteps/ContentTypes/ContentTypes.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react';
+import useFeedSettings from '../../../../hooks/useFeedSettings';
+
+import { useAdvancedSettings } from '../../../../hooks';
+import { CustomCheckbox } from './CustomCheckbox';
+import { useContentTypesOnboarding } from './useContentTypesOnboarding';
+
+export const ContentTypes = (): ReactElement => {
+  const { contentSourceList } = useFeedSettings();
```

Comment:
Why don't you get this all from the same hook?
(EG how `AdvancedSettings` does it now?

Just find the naming a bit odd "expectCommunity" what happens if tomorrow we add another `ContentSource`.
Feel it might be better if we expose `ContentCurationList` and `ContentSourceList` from somewhere in one go?

(Let me know what you think)

Reply:
Seemed like the right approach on my head, but let me give another look to the whole thing to see what makes more sense. Definitely what u're saying makes sense

Reply:
Made a helper function to get the lists based on advanced settings